addons/website_mass_mailing/models/mailing_mailing.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class MassMailing(models.Model):
+    _inherit = "mailing.mailing"
+
+    website_id = fields.Many2one("website", string="Website", help="Website used for the links in your mailing")
```

Comment:
Any reason not to use the `website.multi.mixin` mixin?
If you really need to override the `help` maybe you could still use the mixin and override the field to benefit from the mixin tools

```suggestion
    _inherit = 'mailing.mailing'

    website_id = fields.Many2one('website', string="Website", help="Website used for the links in your mailing")
```
Nitpicking while passing by on this line but it really helps when trying to find things, this python guidelines is really useful, especially for `_name =` and `_inherit =` :+1:

Reply:
You're right :+1: